J/jlqml/build_tarballs.jl

```diff
@@ -48,14 +48,19 @@ install_license $WORKSPACE/srcdir/jlqml*/LICENSE.md
 
 # These are the platforms we will build for by default, unless further
 # platforms are passed in on the command line
-include("../../L/libjulia/common.jl")
-platforms = vcat(libjulia_platforms.(julia_versions)...)
-platforms = expand_cxxstring_abis(platforms)
+platforms_linux = [
+    Platform("aarch64", "linux"; libc="glibc"),
+    Platform("armv7l", "linux"; libc="glibc"),
+    Platform("x86_64", "linux"; libc="glibc"),
+    Platform("i686", "linux"; libc="glibc"),
+    Platform("x86_64", "freebsd"),
+    Platform("powerpc64le", "linux"; libc="glibc"),
+]
+platforms_linux = expand_cxxstring_abis(platforms_linux)
+platforms_win = expand_cxxstring_abis([Platform("x86_64", "windows"), Platform("i686", "windows")])
+platforms_macos = [ Platform("x86_64", "macos"), Platform("aarch64", "macos") ]
 
-filter!(p -> libc(p) != "musl", platforms) # Qt_jll is currently not available for muslc
-# Qt5Declarative_jll is not available for these architectures:
-filter!(p -> arch(p) != "armv6l", platforms)
-filter!(p -> !(arch(p) == "aarch64" && Sys.isapple(p)), platforms)
+platforms = vcat(platforms_linux, platforms_win, platforms_macos)
```

Comment:
Why are you changing the platform computation like this, though, instead of just dropping the `filter!(p -> !(arch(p) == "aarch64" && Sys.isapple(p)), platforms)` line?

Reply:
I see now that this of course discards the julia versions from  the platform, I'll  revert it.